### DISCUSSION TITLE
Bump Node.js version

### DISCUSSION THREAD

**Discussion by fdelucchijr** [nocodb/nocodb#3929] [General]
### Please confirm if bug report does NOT exists already ?

- [X] I confirm there is no existing issue for this

### Steps to reproduce ?

Try to run NocoDB in Oracle A1 Amphere VM’s Kubernetes Cluster. Gets:
```

<--- Last few GCs --->


<--- JS stacktrace --->


#
# Fatal process OOM in insufficient memory to create an Isolate
#

Trace/breakpoint trap (core dumped)
```

This is probably related to [this node issue](https://github.com/nodejs/docker-node/issues/1559).

### Desired Behavior

...

### Project Details

I’ve tried with latest, 0.94,  and 0.80.0 image tags.

**Comment by o1lab** [comment]
Wow, looking at [that issue](https://github.com/nodejs/docker-node/issues/1559). It is definitely beyond our scope to do anything here. NocoDB can run on as little as 256Mb ram on server side.

Plus is it even possible to run a simple nodejs on Oracle's A1 at all ?

**Comment by fdelucchijr** [comment]
> Plus is it even possible to run a simple nodejs on Oracle's A1 at all ?


Yes, currently we are running our server-side in node.js. 
I’ve tried running `node:12` with no success, but `node:14`,`node:14-alpine` and `node:16` works perfectly. There’s any limitation into bumping the node version in the images?
Also, in what I know Node.js 12 is already deprecated... [source](https://azure.microsoft.com/en-us/updates/node12/)

**Comment by o1lab** [comment]
Moving this to discussion.
